core/src/main/java/org/apache/carbondata/core/scan/filter/executer/RawColumnChunkUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Is it related to this PR, if not please raise in separate PR

Reply:
OK